app/action-creators/students.js

```diff
@@ -0,0 +1,102 @@
+import {RECEIVE_STUDENTS,
+  RECEIVE_STUDENT,
+  ADD_STUDENT,
+  DELETE_STUDENT,
+  EDIT_STUDENT,
+  STUDENT_FIRST_NAME,
+  STUDENT_LAST_NAME,
+  STUDENT_EMAIL,
+  STUDENT_CAMPUS,
+} from '../constants';
+import axios from 'axios';
+
+//SYNC ACTIONS
+export const receiveStudent = student => ({
+  type: RECEIVE_STUDENT,
+  student: student,
+});
+
+export const receiveStudents = students => ({
+  type: RECEIVE_STUDENTS,
+  students: students,
+});
+
+export const deleteStudent = id => ({
+  type: DELETE_STUDENT,
+  id: id,
+});
+
+export const createStudent = (campusId, student) => ({
+  type: ADD_STUDENT,
+  student: student,
+});
+
+export const editStudent = (campusId, student) => ({
+  type: EDIT_STUDENT,
+  student: student,
+});
+
+export const setFirstStudentName = (studentFirstName) => ({
+  type: STUDENT_FIRST_NAME,
```

Comment:
Having individual actions/setters for every field on the model seems like it's going to get increasingly tedious as the app grows.

A single `UPDATE_STUDENT` action might be more appropriate here.